pavelib/assets.py

```diff
@@ -384,14 +384,24 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
     """
     Compile CoffeeScript to JavaScript.
     """
-    if not files:
-        files = ["`{}`".format(coffeescript_files())]
+    js_files = ""
+
+    if not coffee_files:
+        coffee_files = ["`{}`".format(coffeescript_files())]
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(coffeescript_files())
+    else:
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(*coffee_files)
```

Comment:
Ah, I didn't realized that we were passing in a bash command to get the coffeescript files. That explains some of the way the code is written now. (Although, we should probably use the `$(...)` syntax rather than the `` syntax, since it nests properly. Also, I'm a bit skeptical of taking an arbitrary bash command in the first place, tbh.